src/main/java/com/skilld/rundeck/plugin/step/kubernetes/KubernetesStep.java

```diff
@@ -105,6 +107,8 @@ public KubernetesStep(final Framework framework) {
 		.property(PropertyUtil.select(RESTART_POLICY, "Restart policy", "The restart policy to apply to the job", true, "Never", Arrays.asList("Never", "OnFailure")))
 		.property(PropertyUtil.integer(COMPLETIONS, "Completions", "Number of pods to wait for success exit before considering the job complete", true, "1"))
 		.property(PropertyUtil.integer(PARALLELISM, "Parallelism", "Number of pods running at any instant", true, "1"))
+		.property(PropertyUtil.string(PERSISTENTVOLUME, "Persistent Volume", "The name of the PVC to use in this job in format <name>;<mountpath>", false, null))
```

Comment:
We could use <name>:<mountpath>;<name>:<mountpath> and so have multiple volumes. Anyway that could be a future enhancement :) (same for secrets)